lib/tempController/tempController.h

```diff
@@ -25,9 +27,44 @@ class TempController {
 
     PID *controller = new PID(255, 0, k_p, k_i, k_d);
 
+  protected:
+    bool humanOverride = false;
+    uint16_t humanSpecifiedValue = 0;
```

Comment:
maybe set the default value to 300 since corresponds w default false value of humanOverride